go.mod

```diff
@@ -54,13 +46,10 @@ require (
 	github.com/tendermint/tendermint v0.32.7
 	github.com/tendermint/tm-db v0.2.0
 	github.com/tyler-smith/go-bip39 v1.0.2 // indirect
-	github.com/wsddn/go-ecdh v0.0.0-20161211032359-48726bab9208 // indirect
-	github.com/xsleonard/go-merkle v0.0.0-20190513010722-059a904a0479
+	github.com/xsleonard/go-merkle v1.1.0
+	golang.org/x/crypto v0.0.0-20200429183012-4b2356b1ed79 // indirect
```

Comment:
why package changes?

Reply:
Actually that are auto updated with new bor version, also giving warnings on build
I had talked to @atvanguard regarding warnings